input/elasticapm/internal/modeldecoder/generator/slice.go

```diff
@@ -142,6 +142,9 @@ func generateJSONPropertySlice(info *fieldInfo, parent *property, child *propert
 	default:
 		return fmt.Errorf("unhandled slice item type %s", itemsType)
 	}
+	if items.Min == "" && strings.HasPrefix(itemType.String(), "uint") {
```

Comment:
Perhaps check if the underlying type is a `*types.Basic`, and use https://pkg.go.dev/go/types#Basic.Info to check if it's unsigned? Something like:

```go
if basic, ok := itemType.Underlying().(*types.Basic); ok {
	if items.Min == "" && (basic.Info() & types.IsUnsigned) != 0 {
		items.Min = json.Number("0")
	}
}
```

Not a big deal though. Feels slightly cleaner to me, but it's a few more lines.